packages/semcom-core/lib/component/models/component-data-types.ts

```diff
@@ -0,0 +1,9 @@
+import { Quad } from 'rdf-js';
+
+export interface ComponentDataTypes {
```

Comment:
Brainstorm: what is the difference in use between this approach and for example the one with an enum used for the event types. Could you pick an optimal one and use it for both cases?

Reply:
I think there is a key difference;
* for the EventType, the enum doesn't do anything else than giving a name to an event, as if it might as well have been a string value
* for DataType, the interface describes what the type of a value must be (not possible with enums)